eng/liveBuilds.targets

```
@@ -174,6 +174,8 @@
         $(LibrariesAllBinArtifactsPath)*.dll;
         $(LibrariesAllBinArtifactsPath)*.pdb"
         IsNative="" />
+      <ExcludeNativeLibrariesRuntimeFiles Condition="'$(IncludeOOBLibraries)' != 'true'"
+                                          Include="$(LibrariesNativeArtifactsPath)/libSystem.IO.Ports.Native.*" />
```

Remove `/` it should already be normalized. 
```suggestion
                                          Include="$(LibrariesNativeArtifactsPath)libSystem.IO.Ports.Native.*" />
```